eox_hooks/actions.py

```diff
@@ -20,6 +21,28 @@
 log = logging.getLogger(__name__)
 
 
+class FakeRequest:
+    """
+    This class represents a fake request, this is needed when the action
+    "trigger_grades_assignment" is executed in an async server.
+    """
+    META = {}
+
+    @staticmethod
+    def is_secure():
+        """
+        Returns a True to mock-up https.
+        """
+        return True
+
+    @staticmethod
+    def get_host():
+        """
+        Returns a string with the site name to mock-up the host.
+        """
+        return settings.SITE_NAME
```

Comment:
This looks great! can you move it to utils? it might make more sense there. What do you think?

Reply:
yes, you are right, this change is done!! thanks a lot.